test/net/sourceforge/kolmafia/textui/parsetree/FunctionTest.java

```diff
@@ -113,4 +121,578 @@ public static Stream<ScriptData> data() {
   public void testScriptValidity(ScriptData script) {
     ParserTest.testScriptValidity(script);
   }
+
+  @Nested
+  class ParamsMatch {
+    // public boolean paramsMatch(final Function that, final boolean base)
+    //
+    // Check if this function's parameters match that function's parameters
+    //
+    // This is used (only) by UserDefineFunction.overridesLibraryFunction
+    // *** This is not tested here (yet)
+
+    // public boolean paramsMatch(final List<? extends TypedNode> params, MatchType match, boolean
+    // vararg)
+    //
+    // Check if a supplied list of values match this function's parameters.
+    //
+    // MatchType,EXACT  - the value and the function parameter must be identical
```

Comment:
`MatchType.EXACT` and same for the others (comma to dot)

Reply:
Fixed.